### ISSUE TITLE
integration test not working in core-dev when running them in terminal

### ISSUE THREAD

**Issue by sagor999** [gitpod-io/gitpod#7953]
### Bug description

You cannot run integration test manually from inside the terminal when your changes are deployed into core-dev. Something to do with permissions (will update once I will try that again)

### Steps to reproduce

Load gitpod in core-dev, try to run integration test

**Comment by sagor999** [comment]
test if it is working with werft /with-vm annotation instead.

**Comment by kylos101** [comment]
> test if it is working with werft /with-vm annotation instead.

Yes, that would be good information, if it does not, we need to let the Platform Team know about supporting that use case.

**Comment by kylos101** [comment]
> You cannot run integration test manually from inside the terminal when your changes are deployed into core-dev

If you can get an copy of the error, I would recommend creating a separate issue and assign it to Platform Team. This way they can triage the k8s permissions.

This way, you could repurpose this issue for asserting whether tests work with harvestor VM preview environments.
